.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+{
+  "image": "ubuntu:22.04",
```

Comment:
I think we should use the ones provided by microsoft?  I generated the devcontainer via vscode and it uses the following image `"mcr.microsoft.com/devcontainers/base:jammy"`

Reply:
I changed the image.